pip/update.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python
+import os
+
+update_list = []
+p = os.popen('sudo -H pip freeze --local',"r")
+while 1:
+    line = p.readline()
+    if not line: break
+    update_list.append(line.rstrip('\n').split('=')[0])
+counter = 0
+while counter < len(update_list):
+        os.system('sudo -H pip install -U "' + update_list[counter] + '"')
+        counter += 1
```

Comment:
You don't use ```counter``` here, so

```python
for entry in update_list:
    subprocess.check_call(['sudo', '-H', 'pip', 'install', '-U', entry])
```

is probably simpler (and more secure, as you're not relying on shell quoting, and hence exposing yourself to security issues from weird package names).